README.md

```diff
@@ -155,6 +155,7 @@ Community Maintained Cars and Features
 | Holden    | Astra 2017<sup>1</sup>        | Adaptive Cruise   | openpilot        | 0mph               | 7mph         |
 | Hyundai   | Elantra 2017-19               | SCC + LKAS        | Stock            | 19mph              | 34mph        |
 | Hyundai   | Elantra 2021                  | SCC + LKAS        | Stock            | 0mph               | 0mph         |
+| Hyundai   | Elantra HEV 2021              | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```

Comment:
hyundai's website uses Hybrid instead of HEV
```suggestion
| Hyundai   | Elantra Hybrid 2021          | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```